common/filters.js

```diff
@@ -14,9 +15,10 @@
     })
 
     // toTitleCase: Capitalizes the first character for each word in a string
-    // TODO: Make it more ~ROBUST~ internationalize this
+    // TODO: Make it more robust by internationalizing this
     // Account for names with slashes, hyphens, Cyrillic names, Klingon, Chinese, etc.
     .filter('toTitleCase', function() {
+        // TODO: Wrap in try/catch
```

Comment:
I don't think this is necessary because if input is not defined, the filter won't run.